react-charts/area-chart/step5.md

```diff
@@ -1,34 +1,56 @@
-Now we'll add tooltips to the area chart...
+Now you will add axis labels to the area chart.
 
 ## Task
 
-1) Make sure the App.js file is still open
+1) Make sure the <strong>App.js file is still open.</strong>
 
-2) Locate the code that looks like the following:
+2) <strong>Locate the code </strong>that looks like the following:
 
 <pre class="file">
 &lt;Chart
+  constrainToVisibleArea
+  containerComponent={&lt;ChartVoronoiContainer labels={({ datum }) =&gt; `${datum.name}: ${datum.y}`} /&gt;}
+  domainPadding={{ x: [30, 25] }}
   height={250}
   width={600}
 &gt;
 </pre>
 
-3) Add the following property to that section:
+3) <strong>Add the following property </strong>to that section:
 
 <pre class="file" data-target="clipboard">
-containerComponent={
-  &lt;ChartVoronoiContainer
-    constrainToVisibleArea
-    labels={({ datum }) =&gt; `${datum.name}: ${datum.y}`}
-  /&gt;
-}
+padding={{
+  bottom: 75,
+  left: 75,
+  right: 50,
+  top: 50
+}}
 </pre>
 
-- The `constrainToVisibleArea` property will alter the position of the tooltip so that it exactly fits within the rendered SVG
-- The `containerComponent` property takes a component instance which will be used to create a container element for standalone charts
-- The `ChartVoronoiContainer` is used to create voronoi overlays for charts, which are useful for attaching events to data points, such as tooltips
+<strong>Note:</strong> This will add more padding to the bottom and left properties to accommodate the axis labels.
 
-4) Once the preview reloads - it should look like this:
-<img src="area-chart/assets/tooltips.png" alt="Chart with tooltips" style="box-shadow: rgba(3, 3, 3, 0.2) 0px 1.25px 2.5px 0px;" />
+4),<strong> Locate the code</strong> that looks like the following:
 
-We'll continue by adding axis labels to the chart in the next step.
+<pre class="file">
+&lt;ChartGroup&gt;
+...
+&lt;/ChartGroup&gt;
+</pre>
+
+5)<strong> Add the following code</strong> above that section:
+
+<pre class="file" data-target="clipboard">
+&lt;ChartAxis label=&quot;Years&quot;/&gt;
+&lt;ChartAxis dependentAxis showGrid label=&quot;Percentage&quot;/&gt;
+</pre>
+
+<strong>Note:</strong> 
+- The grid lines should appear behind the ChartArea components
+- The `Chart` component displays an axis by default, but we've added labels and grid lines

```

Comment:
This is a good place where passive voice makes sense. Let's try, "...but labels and grid lines were added" 

Also, let's add periods after these sentences.